### ISSUE TITLE
DatetimePicker appearance in FF

### ISSUE THREAD

**Issue by andreytank** [vidartf/ipydatetime#14]
I have jupyterlab==2.1.4 with installed ipydatetime==1.2.0
When I'm connecting to JLab through Opera, NativeDatetimePicker and DatetimePicker work as expected
 
---------------
![image](https://user-images.githubusercontent.com/47856746/111476953-b1c37c00-873f-11eb-9e18-c7112b2262b9.png)
![image](https://user-images.githubusercontent.com/47856746/111477029-c3a51f00-873f-11eb-88de-58dd496cb90d.png)

---------------
But when connecting with Mozilla Firefox 86.0, appearance changes to this

---------------
![image](https://user-images.githubusercontent.com/47856746/111477341-1088f580-8740-11eb-9399-d35d208cc1c5.png)
![image](https://user-images.githubusercontent.com/47856746/111477391-1da5e480-8740-11eb-9fb9-7f51c6070f24.png)

---------------
Time section is unclickable, but printable. Until I write down time, value of the widget is a None.

---------------
![image](https://user-images.githubusercontent.com/47856746/111477617-5a71db80-8740-11eb-9d94-9c1affca9612.png)

---------------
Could you please help with this issue? Thank you!

**Comment by vidartf** [comment]
The appearance and behavior is dictated by the browser, as this is a native HTML element. 

> Time section is unclickable, but printable. Until I write down time, value of the widget is a None.

I'm not sure if I follow what you mean here. But to be clear: If you are creating a DatetimePicker, it is completely acceptable for the browser to not give a value until both the time and date has been picked.

Said in another way, can you detail what the current behavior is, contrasted to what you *expect* would happen?

**Comment by andreytank** [comment]
Sure. I expect to get some kind of a widget for date and/or time. In Opera with one click I'm getting one united widget, that allows me to chose both date and time.
In FF I'm getting two separated sections (that's acceptable) for date and time and for a date part I have no question, but time section doesn't work as a widget but as a broken input zone (when clicking on it, I can't see a text cursor, but still can print numbers in it. And the problem is, that when clicking on it, widget doesn't pop up)

**Comment by vidartf** [comment]
So, AFAIK, FF does not support the [datetime-local](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local) input. The module checks for support for that, and falls back to using separate [date](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date) and [time](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/time) controls. If you open the time link in FF, you will see that FF offers no native dialog for picking time.

So in short, this library only aims to expose the native controls of the browser, but the offering of some browsers are quite limited, so it might not be as fully featured as you want.

**Comment by andreytank** [comment]
Thank you for this explanation! Since it is not possible to fix mozilla support from library side, probably I should close this issue then. Thank you again!
